ush/global_cycle.sh

```diff
@@ -269,13 +268,13 @@ CYCLVARS=${CYCLVARS:-""}
 use_ufo=${use_ufo:-.true.}
 DONST=${DONST:-"NO"}
 DO_SFCCYCLE=${DO_SFCCYCLE:-.true.}
-DO_LNDINC=${DO_LNDINC:-.false.}
-DO_SOI_INC=${DO_SOI_INC:-.false.}
-DO_SNO_INC=${DO_SNO_INC:-.false.}
-if [ "$DO_SOI_INC" == ".true." ] || [ "$DO_SNO_INC" == ".true." ] ; then
-        DO_LNDINC=".true."
+DO_LANDINCR=${DO_LANDINCR:-.false.}
```

Comment:
I think you can just do:

`DO_LANDINCR=.false`

as it is set a few lines below. Or you can add an 'else' block.

Reply:
@GeorgeGayno-NOAA I switched to an else block. Thanks